packages/codemods/src/transforms/v3/web-react/__testfixtures__/button-prop-name.input.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+// @ts-ignore: No declaration -- The library is not installed; we don't need to install it for fixtures.
+import { Button } from '@lmc-eu/spirit-web-react';
+
+export const MyComponent = () => (
+  <>
+    <Button color="primary" isSquare>
```

Comment:
isn't the color optional? I don't mind it here, but maybe make the code a bit readable?